src/backend/tests/unit/components/vlmrun/test_vlmrun_transcription.py

```diff
@@ -0,0 +1,171 @@
+import pytest
+from langflow.components.vlmrun.vlmrun_transcription import VLMRunTranscription
+from langflow.schema.data import Data
+
+from tests.base import ComponentTestBaseWithoutClient
+
+
+class TestVLMRunTranscription(ComponentTestBaseWithoutClient):
```

Comment:
Single audio file with mocked client returning segments (assert transcription, metadata.duration, status, prediction_id).

Video flow (ensure video content is included, and optional audio in brackets is appended as implemented).